lib/spree/models/line_item_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module LineItemDecorator
+    protected
```

Comment:
I don't see a reason this shouldn't be `private`.

Reply:
i'll change that